### ISSUE TITLE
[Bug]: sd-civitai-browser-plus continually increase Ram uses when updating tags 

### ISSUE THREAD

**Issue by metapea** [Panchovix/stable-diffusion-webui-reForge#116]
### Checklist

- [ ] The issue exists after disabling all extensions
- [ ] The issue exists on a clean installation of webui
- [X] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [X] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Main details [Here](https://github.com/BlafKing/sd-civitai-browser-plus/issues/318).
When using updating Loras tags/preview images, ram uses keeps going until physical memory is being used at 99% percent and the webui tab goes at 10GB in ram uses when there's a lot of lora info to update. 
Also note this happen in OG forge as well when downloading with the extension. 

### Steps to reproduce the problem

1. Have a large amount of loras
2. Click on loras for update
3. Click on the update button for info and tags

### What should have happened?

Update tags/info/preview images without high ram uses. 

### What browsers do you use to access the UI ?

Mozilla Firefox

### Sysinfo

N/A

### Console logs

```Shell
N/A, No error log
```

### Additional information

> The issue is caused by an extension, but I believe it is caused by a bug in the webui

There was a workaround on reForge's older commits (on the main branch), where if you use `ARG` like `--ui-debug-mode` before starting your checkpoint, updating lora used around 0.5GB of RAM, until i updated to the latest commit where the webui tab keep increasing the ram regardless of if i started the checkpoint.

**Comment by ExissNA** [comment]
Yeah, it's painful, isn't it?  I have to move 3/4 of my resources to a folder that didn't have a link to the webui just to get Forge to be able to load.  Now I'm trying to move to reForge and it's not working so well either.
